This is **part** of the GitHub commit, only the bit fixing the build:

	CMake Error at fem/tests/CMakeLists.txt:17 (FOREACH):
	  FOREACH Invalid integer: 'depth'
	Call Stack (most recent call first):
	  fem/tests/CMakeLists.txt:29 (SUBDIRLIST)


From f02cb33acd59a32c41fe9f67367155e271c22ae8 Mon Sep 17 00:00:00 2001
From: Foad Sojoodi Farimani <foad@Foads-MacBook-Pro.local>
Date: Fri, 3 Apr 2020 22:25:02 +0200
Subject: [PATCH] now compiles on macOS

---
 README.adoc              | 49 ++++++++--------------------------------
 fem/tests/CMakeLists.txt |  2 +-
 2 files changed, 11 insertions(+), 40 deletions(-)

diff --git a/fem/tests/CMakeLists.txt b/fem/tests/CMakeLists.txt
index e8bd7ed84..06d1168ea 100644
--- fem/tests/CMakeLists.txt
+++ fem/tests/CMakeLists.txt
@@ -14,7 +14,7 @@ SET(MESH2D_BIN "${CMAKE_BINARY_DIR}/meshgen2d/src/Mesh2D")
 
 MACRO(SUBDIRLIST result curdir depth)
   set(glob_pattern "*")
-  FOREACH(D RANGE 1 depth)
+  FOREACH(D RANGE 1 ${depth})
     FILE(GLOB children RELATIVE ${curdir} ${glob_pattern}) 
     FOREACH(child ${children})
       IF(IS_DIRECTORY ${curdir}/${child})
